openthread-2018-02-27/build.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
why this licence header is so sort
I used to ones like this: https://github.com/google/libprotobuf-mutator/blob/master/src/mutator_test.cc

Reply:
Looks like we should be using the longer header.

@kcc: What was the reason for using a short header?

Maybe in a future PR I should update the headers.